plugins/lookup/passwordstore.py

```diff
@@ -60,19 +60,46 @@
         description: use alphanumeric characters.
         type: bool
         default: 'no'
+      missing:
+        description: List of preference about what to do if the password file is missing.
+        version_added: 3.1.0
+        type: str
+        default:
+          - error
+        choices:
+          - error
+          - warn
+          - empty
+          - create
 '''
 EXAMPLES = """
 # Debug is used for examples, BAD IDEA to show passwords on screen
 - name: Basic lookup. Fails if example/test doesn't exist
   ansible.builtin.debug:
     msg: "{{ lookup('community.general.passwordstore', 'example/test')}}"
 
+- name: Basic lookup. Warns if example/test does not exist and returns empty string
+  ansible.builtin.debug:
+    msg: "{{ lookup('community.general.passwordstore', 'example/test missing=warn')}}"
+
 - name: Create pass with random 16 character password. If password exists just give the password
   ansible.builtin.debug:
     var: mypassword
   vars:
     mypassword: "{{ lookup('community.general.passwordstore', 'example/test create=true')}}"
 
+- name: Create pass with random 16 character password. If password exists just give the password
```

Comment:
I know that you have put it like this because of other examples, but this is clearer IMHO.
Also on creating random 16 character, I know that this has also been like this in other examples, but is this the default on all platforms? Hasn't some distribution changed this default behaviour for whatever reason they may have? Also `pass` may change its default behaviour. So why should it be explicitly stated here?
```suggestion
- name: Create pass with default configurations of system. Returns password if it exists
```

Reply:
The default for the `length` option to this lookup is 16, so it's independent of the `pass` version.

Reply:
I just copied the description. As I did not change the behavior in this case I did not changed the description either. Yes, the default is 16 and it is platform independent as it is not generated by the `pass` but by the plugin itself. The default value for `length` parameter is 16.